tests/pages/pmmSettingsPage.js

```diff
@@ -90,73 +90,93 @@ module.exports = {
   },
   tooltips: {
     dbaas: {
+      iconLocator: locate('$advanced-dbaas').find('div[class$="-Icon"]'),
       text: 'Option to enable/disable DBaaS features. Disabling DBaaS does not suspend or remove running clusters.',
       link: links.dbaasDocs,
     },
     stt: {
+      iconLocator: '//div[@data-testid="advanced-stt"]//div/div/div/div',
```

Comment:
any reason why not using locate here?

Reply:
I've moved this locator from: https://github.com/percona/pmm-ui-tests/blob/main/tests/pages/pmmSettingsPage.js#L218

Would you like me to update it using `locate()` ?